src/main/javascript/src/components/ListParticipants.vue

```diff
@@ -84,7 +85,23 @@ export default {
       this.tempSelectedInAGroup = [];
 
       this.moveToHandler(option, tempSelectedInAGroup);
+    },
+        handleOnChange(value) {
+      if (value === 0) {
```

Comment:
You could simplify this conditional statement to a ternary operator like so:
`this.tempSelectedInAGroup = (value===0) ? this.listOfParticipants : []`